### ISSUE TITLE
Permanence of object and device movement tracking

### ISSUE THREAD

**Issue by Wolgraph** [AR-js-org/AR.js#166]
Hi. Second question I posted here.
I'm trying to figure out if AR.js support in any way device or environment tracking.
My real scene (a glass case of a museum) where to overlay the AR objects is too big to have the marker constantly tracked.
Thus I need my web script to be able to use the marker to correctly place the objects and then the device or environmental tracking system keep them in place.

**Comment by nicolocarpignoli** [comment]
No, unfortunately it's not something that AR.js supports for now and we do not have any ETA for this.
I suggest you to try to integrate Model-Viewer (by Google but OSS) on your web app, for this feature.

**Comment by Wolgraph** [comment]
I was thinking about trying to integrate opencv. js to track the environment and mobile sensor (gyroscope and accelerometer) as support to limit computation errors. This way Ar.js will place the object ans the other libraries will control over the objects' tansforms. 
It may be a little naive, but I hope to male it work just fine.

**Comment by AdamVI2** [comment]
**Wolgrpah**, could you please share your results if you make it work? There have been more people interested in that feature out there I believe.

**Comment by Wolgraph** [comment]
If I find something, I'll post it here.
I'm trying to study opencv as fast as I can, but I never studied numeric calculus at the University, thus computer vision is quite hard to learn in few days 😅

**Comment by bonisaputra** [comment]
i'm trying to make these feature too, but for now, movement only tracked on marker AR, but unfortunately doesnt work in markerless AR. lets solve this
